ansible_role/operator/Dockerfile-olm

```diff
@@ -0,0 +1,8 @@
+FROM quay.io/water-hole/ansible-operator
+
+COPY . /opt/ansible/roles/automation-broker
+COPY playbooks/operator.yml /opt/ansible/deploy.yaml
+COPY operator/watches.yaml /opt/ansible/watches.yaml
+
+RUN sed -i "s/olm_managed: false/olm_managed: true/" \
```

Comment:
Can we parameterize this on the main dockerfile or is this the standard way to handle this?